examples/neighbors/plot_nca_classification.py

```diff
@@ -40,7 +40,7 @@
     X, y, stratify=y, test_size=0.7, random_state=42
 )
 
-h = 0.01  # step size in the mesh
+h = 0.015  # step size in the mesh
```

Comment:
I think this is more than enough. The boundary is still rough but this is completely fine.

```suggestion
h = 0.05  # step size in the mesh
```